bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/handler/ShellyBaseHandler.java

```diff
@@ -293,8 +293,11 @@ public boolean initializeThing() throws ShellyApiException {
         cache.clear();
         resetStats();
 
-        logger.debug("{}: Start initializing for thing {}, type {}, IP address {}, Gen2: {}, CoIoT: {}", thingName,
-                getThing().getLabel(), thingType, config.deviceAddress, gen2, config.eventsCoIoT);
+        profile.initFromThingType(thingType);
+        logger.debug(
+                "{}: Start initializing for thing {},  type {}, Device address {}, Gen2: {}, isBlu: {}, alwaysOn: {}, hasBattery: {}, CoIoT: {}",
+                thingName, getThing().getLabel(), thingType, config.deviceAddress.toUpperCase(), gen2, profile.isBlu,
```

Comment:
For a debug message, wouldn't it be better to have the actual value rather than an uppercased version which can be derived from the original anyway?
```suggestion
                thingName, getThing().getLabel(), thingType, config.deviceAddress, gen2, profile.isBlu,
```

Reply:
the value itself works in upper/lower-case, but I think uppercase makes HEX more readable

Reply:
OK. My point was the it's a debug message, so it seemed logical to have as much information as possible, and by converting to uppercase you are losing information. But it's a minor comment, it's also fine as is.